lsp-mode.el

```diff
@@ -683,6 +697,11 @@ If this is set to nil, `eldoc' will show only the symbol information."
                   (const :tag "Include document symbols if server supports it." symbols)))
   :group 'lsp-mode)
 
+(defcustom lsp-headerline-breadcrumb-enable-symbol-numbers t
```

Comment:
I'm not sure if this should be `t` by default, but it seems a simple UI change so I'm ok with this
![image](https://user-images.githubusercontent.com/7820865/87465956-d92fdb00-c5eb-11ea-9fc6-c1ff86b60fbf.png)

Reply:
I vote this to be off by default - the central feature of the headerline is the visual function, not the navigation one. Thus IMO it should be optimized for better visuals, not for navigation.